fapi-client/src/main/scala/com/gu/facia/api/models/curatedcontent.scala

```diff
@@ -4,56 +4,47 @@ import com.gu.contentapi.client.model.Content
 import com.gu.facia.api.utils.{ResolvedMetaData, ItemKicker}
 import com.gu.facia.client.models.{SupportingItem, MetaDataCommonFields, Trail, TrailMetaData}
 
-case class ImageReplace(imageSrc: String, imageSrcWidth: String, imageSrcHeight: String)
-
-object ImageReplace {
-  def fromTrailMeta(trailMeta: MetaDataCommonFields): Option[ImageReplace] =
-    for {
-      imageReplace <- trailMeta.imageReplace.filter(identity)
-      imageSrc <- trailMeta.imageSrc
-      imageSrcWidth <- trailMeta.imageSrcWidth
-      imageSrcHeight <- trailMeta.imageSrcHeight
-    } yield ImageReplace(imageSrc, imageSrcWidth, imageSrcHeight)
-}
+case class FaciaImage(
+  imageType: String,
```

Comment:
I am happy with a String for this.
We need to know the type of the image, because we can implement different behavior base on this type.